docs/_includes/about.html

```diff
@@ -1,7 +1,7 @@
 <!-- About -->
     <div class="about">
       
-  <section class="page-section" id="{{ site.data.sitetext.about.section | default: "about" }}">
+  <section class="page-section" style="padding-bottom: 70px;" id="{{ site.data.sitetext.about.section | default: "about" }}">
```

Comment:
@Neilblaze what do you think about including this style update inside of `[_about.scss](https://github.com/layer5io/service-mesh-performance/blob/master/docs/_sass/layout/_about.scss)` instead of inline here?

Reply:
Actually what I noticed is that, all the styling is being post-converted to css from sass. Since the padding between each sections kind of looked a bit large, so I thought to reduce them a little bit by adding that inline css. Since inline css has the higher priority, it makes the required changes.